packages/frontend/store/system/reducers.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  GET_WIDTH,
+  IWindowPropsState,
+  IGetWidthAction
+} from './types';
+
+const initialState: IWindowPropsState = {
+  windowProps: {
+    width: 1200
```

Comment:
Should this really be set? Or is this the default value so that SSR NextJS doesn't complain?

Reply:
I'm not actually setting window width with this reducer, only getting it from window.innerWidth whenever the user resizes. The initial state is never used.

How can I get the initial width if I don't have access to window except in useEffect?

Reply:
`useEffect` will run when the component mounts I believe, so the initial load up will always trigger your function. I believe by default if you give it an initial state, it will actually set that into the state. I guess the moment the window loads it will update, but it might cause flash issues if set to an inappropriate number (E.g. PC view displaying on a mobile phone on initial load). I'm not sure if this will be a problem.

Also, we should just have a state variable like `mobileView: false`, instead of the actual width. We don't want the components to do the calculations of when they should be in mobile view or not. It should be centralised and decided in one place (I think _app.tsx is good, or write it in a separate file and import it into _app.tsx).